src/client/index.js

```diff
@@ -36,6 +37,9 @@ function Client(options) {
 	if (options.syncCssVariable) {
 		this.use(syncCssVariable);
 	}
+	if (options.syncFlags) {
+		this.use(syncFlags);
```

Comment:
I think an argument could be made that this doesn't need to be an option and it's just something that's always ON?

Reply:
I left it as an option in cause there were a lot of flags to sync and it could effect performance. I'm fine with turning it on though since I think we have a total of 3 client side flags right now.

Reply:
Yeah I think the number is always going to be pretty small, and it's possible that some web components (like HTML editor) might want to use flags one day... and they don't really have the ability of turning this option on so I think just always having it on is good.